ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -158,11 +159,20 @@ class FlaskInstrumentor(BaseInstrumentor):
 
     def __init__(self):
         super().__init__()
-        self._original_flask = None
+        self._original_flask_class = None
 
-    def _instrument(self):
-        self._original_flask = flask.Flask
+    def _instrument(
+        self, flask_class=None
```

Comment:
any reason not to use kwargs here? and in `_uninstrument`?

Reply:
yes, only because `kwargs` are not used in `_instrument` nor in `_uninstrument`.

Reply:
but you could pull `flask_class` from the `kwargs` instead of overriding the method signature correct? any reason not to go that route?

Reply:
we could do that but it may not play nice with the documentation of the optional `flask_class` argument (not that there is any here, though :sweat_smile:)